markdown/report.Rmd

```diff
@@ -166,121 +241,164 @@ More recently in December 2016 the ["Health and Social Care Delivery Plan"](http
+The NHS Board where residents spent the highest percentage of time in their last six months of life at home or in the community was `r max_hb$hb` where `r max_hb$qom`% of an individual’s last six months of life was spent at home or in a community setting and `r max_hb$qom_hosp`% in hospital. On average, this equates to approximately `r english(round_half_up(182.5 * (max_hb$qom_hosp / 100)))` days spent in a hospital setting within the last six months of life. 
+
+The NHS Board with the lowest percentage of time on average spent at home or in a community setting in their last six months of life was `r min_hb$hb` where `r min_hb$qom`% of an individual’s last six months of life was spent at home or in a community setting and `r min_hb$qom_hosp`% in hospital. This equates to approximately `r round_half_up(182.5 * (min_hb$qom_hosp / 100))` days spent in a hospital setting within the last six months of life. The variation in percentages across Scotland could be affected by the age profile and rurality of the different health boards.
 
-The NHS Board with the lowest percentage of time on average spent at home or in a community setting in their last six months of life was NHS Borders where 86.4% of an individual’s last six months of life was spent at home or in a community setting and 13.6% in hospital. This equates to approximately twenty five days spent in a hospital setting within the last six months of life. The variation in percentages across Scotland could be affected by the age profile and rurality of the different health boards.
+Note that hospital data are provisional for `r latest$fy`. This publication will be updated in October `r year(pub_date)` with final figures. _More detail of completeness here_. See [Appendix 2](#Appendix2) for more information.
 
-Note that hospital data are provisional for 2018/19. This publication will be updated in October 2019 with final figures.  Figures for the latest year should therefore be treated with caution particularly NHS Greater Glasgow & Clyde and NHS Forth Valley. There could be an undercount of hospital admissions, particularly for people who died in the last three months of the year. See [Appendix 2](#Appendix2) for more information.
+##### Figure 2: Map showing the percentage of last six months of life spent at home or in the community in `r latest$fy`^1^; by NHS Board of Residence^2,3^
 
 ##### Map HERE
 
 <div custom-style = "Normal_Notes">
-1. Figures for 2018/19 are incomplete and will be revised in October 2019 publication. See [Appendix 2](#Appendix2) for more information.
+1. Figures for `r latest$fy` are incomplete and will be revised in October `r year(pub_date)` publication. See [Appendix 2](#Appendix2) for more information.
 2. The Health Board of Residence is the usual place of residence prior to death.
 3. Health Boards are based on the NHS Board boundaries which came into effect 1st April 2014.
```

Comment:
Shall we claim the health board version as 1st April 2019 rather than 2014 due to 2019 council area boundary change?

Reply:
Happy to take your advice on this, although I didn't think that the NHS board boundaries had changed?

Reply:
The health board boundaries do change because the council area boundaries changed. HB boundaries align with CA boundaries. There is no population change for 2018 as the boundary change was very small. but as you know for 2019 there was population change as well (impact nearly 400 people).